.github/workflows/gradle.yml

```diff
@@ -77,7 +77,7 @@ jobs:
           gradle-distribution-sha-256-sum-warning: false
           concurrent: true
           # TODO: be able to remove the skipSlowTests property
-          arguments: --scan check -DskipSlowTests=true
+          arguments: --scan check :airbyte-cdk:bulk:bulkCdkIntegrationTest -DskipSlowTests=true
```

Comment:
Why add this here? There are no other integration tests normally in that check?

Reply:
this is (afaik) the only pr check that actually exercises cdk code? 

and these "integration tests" aren't slow to run, so I feel ok with just adding them here. Could be convinced to make a new pr check entirely for bulk CDK stuff, if you think that would be clearer

Reply:
Partly, it feels weird since `check` is a stage and `:airbyte-cdk:bulk:bulkCdkIntegrationTest` is a specific task.

Like if this is what we want, then maybe just make check dependent on this task?

Reply:
done in https://github.com/airbytehq/airbyte/pull/45634/commits/b8aa176bde53d9eda15c008cd8a6df181a58c9d4